### ISSUE TITLE
Sign up problem

### ISSUE THREAD

**Issue by zipyazilim** [adrianhajdin/banking#3]
I copied the files from github but the Registration form is not working. Is everyone the same?

**Comment by nizar787** [comment]
yes same problem even at the deployed website   https://banking-jet.vercel.app/sign-up   , the signup is not working

**Comment by fasakinhenry** [comment]
What do we now do? I followed through the course and I was not able to get my version to work
